sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -331,8 +331,8 @@ class JoinSuite extends QueryTest with SharedSQLContext {
   }
 
   test("full outer join") {
-    upperCaseData.where('N <= 4).registerTempTable("left")
-    upperCaseData.where('N >= 3).registerTempTable("right")
+    upperCaseData.where('N <= 4).registerTempTable("`left`")
```

Comment:
That is what we currently do. You can use any keyword as long as you wrap it with backticks.